### DISCUSSION TITLE
Push events ro webdav

### DISCUSSION THREAD

**Discussion by roughnecks** [bitfireAT/icsx5#336] [Q&A] [answered]
Hello,
can icsx5 push events to the webdav server where ics calendar is hosted?
or is it a one way (pull) only?

Thanks

**Comment by ArnyminerZ** [comment, marked as answer]
No, sorry, ICSx⁵ is only for downloading data. If you want to push data to a CalDAV server, we also have [DAVx⁵](https://github.com/bitfireAT/davx5-ose/), maybe that's useful to you.

**Reply by roughnecks** [reply]
Okay 👍
